core/templates/pages/contributor-dashboard-admin-page/contributor-admin-dashboard-page.component.html

```diff
@@ -84,25 +84,15 @@ <h2 class="stat-value-with-rating e2e-test-oppia-total-plays">
           </div>
         </div>
         <div class="filter-sub-container">
-          <div class="last-activity-selector-container">
-            <div #activityDropdown  class="oppia-filter-selector-container e2e-test-last-activity-selector"
-                 (click)="toggleActivityDropdown()">
-              <div class="oppia-filter-selector-inner-container e2e-test-last-activity-selector-selected">
-                <b>Filter by last activity: </b>
-                   {{ selectedLastActivity ? 'Within past ' + selectedLastActivity + ' days' : 'All time' }}
-                <span class="fas fa-caret-down oppia-filter-selector-inner-container-arrow"
-                      [@lastActivityDropdown]="activityDropdownShown ? 'expanded' : 'collapsed'">
-                </span>
-              </div>
-              <div *ngIf="activityDropdownShown"
-                   class="oppia-filter-selector-dropdown-container e2e-test-last-activity-selector-dropdown">
-                <div class="oppia-filter-selector-dropdown-option e2e-test-last-activity-selector-option"
-                     *ngFor="let option of lastActivity"
-                     (click)="selectLastActivity(option)"
-                     [ngClass]="{'oppia-filter-selector-dropdown-option-selected' : option === selectedLastActivity}">
-                  {{ option ? 'Within past ' + option + ' days' : 'All time' }}
-                </div>
-              </div>
+          <div class="date-selector-container">
+            <div class="date-selector-inner-container">
+              <b *ngIf="isSubmitterTab()" class="filter-label">Filter by Last Contributed after:</b>
+              <b *ngIf="isReviewerTab()" class="filter-label">Filter by Last Reviewed after:</b>
+              <mat-form-field>
+                <input matInput class="e2e-test-last-date-picker-input" [matDatepicker]="lastDatepicker" (dateChange)="changeLastDate($event.value)" [(ngModel)]="lastDateToFilterUsersActivity" [max]="today">
```

Comment:
lastDatepicker -> lastDatePicker. Otherwise it seems like a last datepicker, not last date picker

Reply:
Done